hypha/static_src/tailwind/components/strapline.css

```diff
@@ -0,0 +1,37 @@
+.strapline {
+  @apply mb-8 text-fg-muted text-pretty;
+
+  h1 {
+    font-size: var(--h1-size);
+    @apply leading-tight;
+  }
```

Comment:
Styles copied from https://github.com/torchbox/wtsh-hypha/blob/d252e7461e77f44ead7c9bd97932b712730b6b3c/hypha/static_src/tailwind/utilities/typography.css#L29-L32 (couldn't figure out how to `@apply` them)